nodejs/eks/examples/aws-profile/package.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "example-aws-profile",
+    "devDependencies": {
+        "typescript": "^3.0.0"
+    },
+    "dependencies": {
+        "@pulumi/pulumi": "^1.0.0",
```

Comment:
Should we bump this to `2.0.0`, or do you want to hold off on that?

Reply:
Good catch!